athena-neptune/src/main/java/com/amazonaws/athena/connectors/neptune/propertygraph/rowwriters/BaseRowWriter.java

```diff
@@ -0,0 +1,36 @@
+/*-
+ * #%L
+ * athena-neptune
+ * %%
+ * Copyright (C) 2019 Amazon Web Services
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.amazonaws.athena.connectors.neptune.propertygraph.rowwriters;
+
+/**
+ * This class is a Utility class to create Extractors for each field type as per
+ * Schema
+ */
+public class BaseRowWriter 
```

Comment:
was this committed by accident? seems incomplete

Reply:
this was just being used to have enums defined at base class. created Enums.java for all enums

 https://github.com/abhishekpradeepmishra/aws-athena-query-federation/blob/athena-neptune-edgesupport/athena-neptune/src/main/java/com/amazonaws/athena/connectors/neptune/propertygraph/Enums.java